### PR TITLE
Update azure_rm_roledefinition related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_roledefinition related document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_roledefinition/azure_rm_roledefinition_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by Fred-sun** [comment]
ready_for_review

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
shipit

File: lib/ansible/modules/cloud/azure/azure_rm_roledefinition.py [hunk 1]
```diff
@@ -48,26 +48,28 @@
                     - List of denied data actions.
                 type: list
     assignable_scopes:
-        description: List of assignable scope of this definition.
+        description:
+            - List of assignable scope of this definition.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_roledefinition.py]:
```suggestion
            - List of assignable scopes of this definition.
```

File: lib/ansible/modules/cloud/azure/azure_rm_roledefinition.py [hunk 2]
```diff
@@ -48,26 +48,28 @@
...
-      description:
-        - Assert the state of the role definition.
-        - Use 'present' to create or update a role definition and 'absent' to delete it.
-      default: present
-      choices:
-        - absent
-        - present
+        description:
+            - Assert the state of the role definition.
+            - Use C(present) to create or update a role definition and C(absent) to delete it.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_roledefinition.py]:
```suggestion
            - Use C(present) to create or update a role definition; use C(absent) to delete it.
```

File: lib/ansible/modules/cloud/azure/azure_rm_roledefinition.py [hunk 3]
```diff
@@ -87,7 +89,8 @@
 
 RETURN = '''
 id:
-    description: Id of current role definition.
+    description:
+        - Id of current role definition.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_roledefinition.py]:
```suggestion
        - ID of current role definition.
```

File: lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py [hunk 4]
```diff
@@ -58,60 +60,64 @@
...
-                sample: Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read
-            not_data_actions:
-                description:
-                    - List of denied actions.
-                returned: always
-                type: list
-                sample: Microsoft.Storage/storageAccounts/blobServices/containers/blobs/write
+        id:
+            description:
+                - Role Definition id.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py]:
```suggestion
                - Role Definition ID.
```

File: lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py [hunk 5]
```diff
@@ -58,60 +60,64 @@
...
+            sample: myCustomRoleDefinition
+        name:
+            description:
+                - System assigned role name.
+            returned: always
+            type: str
+            sample: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+        assignable_scopes:
+            description:
+                - List of assignable scope of this definition.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py]:
```suggestion
                - List of assignable scopes of this definition.
```

File: lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py [hunk 6]
```diff
@@ -58,60 +60,64 @@
...
+            returned: always
+            type: list
+            sample: [ "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myResourceGroup" ]
+        permissions:
+            description:
+                - List of Role Definition peremissions.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py]:
```suggestion
                - List of Role Definition permissions.
```

File: lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py [hunk 7]
```diff
@@ -58,60 +60,64 @@
...
+                    sample: [ 'Microsoft.Compute/virtualMachines/write' ]
+                data_actions:
+                    description:
+                        - List of allowed data actions.
+                    returned: always
+                    type: list
+                    sample: [ 'Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read' ]
+                not_data_actions:
+                    description:
+                        - List of denied actions.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py]:
```suggestion
                        - List of denied data actions.
```

**Review by acozine — COMMENTED** [review]
Thanks @Fred-sun. Overall looks good. I found a couple of nits to pick (typos, capitalization, plurals). If you're okay with those changes, I'll merge this tomorrow.

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py [hunk 8]
```diff
@@ -58,60 +60,64 @@
...
+            type: list
+            sample: [ "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myResourceGroup" ]
+        permissions:
+            description:
+                - List of Role Definition permissions.
+            returned: always
+            contains:
+                actions:
+                    description:
+                        - List of allowed data actions.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py]: There are four parts to the permissions: `actions`, `not_actions`, `data_actions`, and `not_data_actions`. 

The description of `actions` should be just `List of allowed actions.`

```suggestion
                        - List of allowed actions.
```

File: lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py [hunk 9]
```diff
@@ -58,60 +60,64 @@
...
+                    returned: always
+                    type: list
+                    sample: [ 'Microsoft.Compute/virtualMachines/read' ]
+                not_actions:
+                    description:
+                        - List of denied data actions.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py]: This one is `not_actions`, so no `data` here either.

```suggestion
                        - List of denied actions.
```

File: lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py [hunk 10]
```diff
@@ -58,60 +60,64 @@
...
+                    returned: always
+                    type: list
+                    sample: [ 'Microsoft.Compute/virtualMachines/write' ]
+                data_actions:
+                    description:
+                        - List of allowed data actions.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py:111]: This one is correct.

File: lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py [hunk 11]
```diff
@@ -58,60 +60,64 @@
...
+                    returned: always
+                    type: list
+                    sample: [ 'Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read' ]
+                not_data_actions:
+                    description:
+                        - List of denied data actions.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py:117]: This one is also correct.

**Review by acozine — COMMENTED** [review]
Oops. Include `data` in the descriptions of only `data_actions` and `not_data_actions`.

**Comment by Fred-sun** [comment]
@acozine Thanks for your review! update new!

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by acozine** [comment]
Looks good, thanks @Fred-sun.

**Pull request merged**